src/OrbitGl/TimeGraph.cpp

```diff
@@ -887,6 +895,13 @@ void TimeGraph::DoDraw(orbit_gl::PrimitiveAssembler& primitive_assembler,
   if (layout_->GetDrawTimeGraphMasks()) {
     DrawMarginsBetweenChildren(primitive_assembler);
   }
+
+  // Track header sizer.
+  float sizer_width = layout_->GetSpaceBetweenTracks();
+  float header_width = layout_->GetTrackHeaderWidth();
+  track_header_sizer_->SetWidth(sizer_width);
+  track_header_sizer_->SetHeight(GetHeight());
+  track_header_sizer_->SetPos(GetPos()[0] + header_width - sizer_width, GetPos()[1]);
```

Comment:
`DoDraw` and `DoUpdatePrimitives` must not call any methods that result in a redraw. Move this to `TimeGraph::DoUpdateLayout`.

Reply:
Done